Makefile

```diff
@@ -213,18 +213,21 @@ install: build
 	$(SILENT)cp -r packages $(ponydir)/
 ifeq ($(symlink),yes)
 	@mkdir -p $(prefix)/bin
-	@mkdir -p $(prefix)/lib/$(arch)
```

Comment:
doesnt this removal reintroduce the problem we identified in zulip the other day?

Reply:
No, because we would no longer be placing the symlinks in the `$(arch)` sub-directory. So it would not need to be created